ecs-cli/modules/utils/utils.go

```diff
@@ -139,12 +142,14 @@ func ParseLoadBalancers(flagValues []string) ([]*ecs.LoadBalancer, error) {
 				ContainerName:  aws.String(m["containerName"]),
 				ContainerPort:  aws.Int64((containerPort)),
 			})
-		} else {
+		} else if elbv1 {
 			list = append(list, &ecs.LoadBalancer{
 				LoadBalancerName: aws.String(m["loadBalancerName"]),
 				ContainerName:    aws.String(m["containerName"]),
 				ContainerPort:    aws.Int64((containerPort)),
 			})
+		} else {
+			return nil, fmt.Errorf("InvalidParameterException: Target Group Arn or Load Balancer Name can not be blank")
```

Comment:
Just `Target Group Arn or Load Balancer Name can not be blank`. "InvalidParameterException" is the error code defined by aws sdk go.